### DISCUSSION TITLE
[search] change the default message if not result found

### DISCUSSION THREAD

**Discussion by foutesting** [fomantic/Fomantic-UI#1721] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Hi,
I do a search with the module $('.') 
  .search({ })
If there are no results, 2 errors messages are displayed by default :
"No Results"
"Your search returned no results"

Is there a way to change these messages using a method ?

Thank

**Comment by lubber-de** [comment, marked as answer]
Yes, you can override the texts
```javascript
$('#mysearchbox').search({
  error: {
    noResults: 'Nothing to see here, go ahead', // was "Your search returned no results" before
    noResultsHeader: 'Hey, what did you do???', // was "No Results" before
  }
})
```
See
https://fomantic-ui.com/modules/search.html#/settings
![image](https://user-images.githubusercontent.com/18379884/96624824-9b8ace80-130d-11eb-8a96-7b6c082e2326.png)

**Comment by foutesting** [comment]
I did a test. Nothing changes. Always the same default messages

**Reply by foutesting** [reply]
sorry. Great. It work. Thank
